packages/vega-crossfilter/src/Bitmaps.js

```diff
@@ -24,11 +23,15 @@ export default function Bitmaps() {
       }
     },
 
-    remove(num, map) { // map: index -> boolean (true => remove)
-      const n = data.length,
-            copy = Array(n - num),
-            reindex = data; // reuse old data array for index map
-      let t, i, j;
+    remove(num, map) {
+      // map: index -> boolean (true => remove)
+      const n = data.length;
+
+      const copy = Array(n - num);
+      const reindex = data; // reuse old data array for index map
+      let t; // reuse old data array for index map
```

Comment:
@lgrammel this looks like a bug in the conversion.

Reply:
Thanks - taking a look now

Reply:
I've provided a fix in P42. It's still processing for a few repos, but maybe it has updated suggestions already for vega (i.e. is not in progress any more).

Reply:
I've reviewed the PR and found a few more occurrences of the bug. I've validated the fix with a fresh fork of Vega - for all known occurrences the problem is gone.

Reply:
Thank you. Here is a new pull request: https://github.com/vega/vega/pull/3126